examples/tools/scene_viewer.rs

```diff
@@ -286,11 +286,7 @@ fn camera_spawn_check(
                     &transform.back(),
                     perspective_projection.far(),
                 );
-                let camera = Camera {
-                    near: perspective_projection.near,
-                    far: perspective_projection.far,
-                    ..default()
-                };
+                let camera = Camera::default();
```

Comment:
We should just delete this line and let the `..default()` in PerspectiveCameraBundle fill in the camera value.